src/modules/currency/currency.graphql.js

```diff
@@ -1,13 +1,13 @@
 const currencyType = `type Currency {
     _id: ID!
-    date: String!
+    date: String
     convertOptions: [ConvertOption!]
 }
 `;
 
 const currencyInput = `
     input CurrencyInput {
-    date: String!
+    date: String
```

Comment:
why it is not required anymore?

Reply:
we do not have to enter the date manually

Reply:
What is this date about? What real-world thing does it reflect?